docs/Way-of-Working/Code-Reviews.md

```diff
@@ -0,0 +1,36 @@
+# Code reviews
+
+The purpose of a code review is to ensure the quality, readability, and that all requirements from the ticket have been
+met for a change before it gets merged into the main codebase. Additionally, code reviews are a communication tool, they
+allow team members to stay aware of changes being made.
+
+Code reviews involve having a team member examine the changes made by another team member and give feedback or ask
+questions if needed.
+
+## Creating a Pull Request
+
+We use GitHub pull requests (PR) for code reviews. You can make a draft PR if your work is still in progress. When you
+are done you can remove the draft status. A team member may start reviewing if the PR does not have a draft status.
+
+For team ADRs at least 3 accepting reviews are required. or all team members should accept if it can be expected that
+the ADR is controversial.
```

Comment:
Can we know in advance is an ADR is controversial? ;) And is it possible per PR to request approval from all members or is this not a rule we can enforce technically?

Reply:
No, i think a controversial changes is up to the reviews/author 

I do not think we can enforce it technically.

Reply:
Indeed. The point is it you EXPECT it the be controversial you should include more people

Reply:
Check!